modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/info/item/provider/ObjectEntryInfoItemFormProvider.java

```diff
@@ -515,7 +516,19 @@ private String _getRelationshipLabelFieldName(ObjectField objectField) {
 			return "id";
 		}
 
-		return titleObjectField.getName();
+		String titleObjectFieldName = titleObjectField.getName();
+
+		if (titleObjectFieldName.equals("createDate")) {
+			return StringUtil.replace(
+				titleObjectFieldName, "createDate", "dateCreated");
+		}
+
+		if (titleObjectFieldName.equals("modifiedDate")) {
+			return StringUtil.replace(
+				titleObjectFieldName, "modifiedDate", "dateModified");
+		}
+
+		return titleObjectFieldName;
```

Comment:
@pedro-oliveira446 great job!

Could you please check if we can simplify to:

```
if (titleObjectField == null) {
	return "id";
}
else if (Objects.equals(titleObjectField.getName(), "createDate")) {
	return "dateCreated";
}
else if (Objects.equals(titleObjectField.getName(), "modifiedDate")) {
	return "dateModified";
}

return titleObjectField.getName();
```

Thanks!

Reply:
@carolmariaabb Apparently it's working properly.

Thanks.